src/modules/lua/engine_lua.c

```diff
@@ -0,0 +1,542 @@
+#include "../../valkeymodule.h"
```

Comment:
We should add this copyright header to new files:

```suggestion
/*
 * Copyright (c) Valkey Contributors
 * All rights reserved.
 * SPDX-License-Identifier: BSD-3-Clause
 */
#include "../../valkeymodule.h"
```